setup.py

```diff
@@ -1,4 +1,11 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
 
+from future import standard_library
```

Comment:
@rorygrandin this will break running your setup.py for people without future yet